routing/src/event.rs

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
+
+//! Generic thread-local event logs. See the tests for sample usage
+
+#![allow(unused)]
+
+use chrono::DateTime;
+use chrono::Local;
+use std::cell::RefCell;
+use std::fmt::Display;
+
+/// An `Event` is a wrapper over a generic type T that represents something that happened.
```

Comment:
```suggestion
/// An [`Event`] is a wrapper over a generic type T that represents something that happened.
```

Rustdoc mark-up? (Also in the other Rustdoc comments)